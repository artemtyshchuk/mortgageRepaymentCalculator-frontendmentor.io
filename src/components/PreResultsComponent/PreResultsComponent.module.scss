.preResultsComponent {
  position: relative;
  top: 20%;
  display: grid;
  justify-items: center;
}

.preResultsTitle {
  margin-top: 16px;
  margin-bottom: 0;
  color: var(--color-white, #fff);
  text-align: center;

  /* desktop/text-preset-2 */
  font-family: "Plus Jakarta Sans";
  font-size: var(--font-size-text-preset-2, 24px);
  font-style: normal;
  font-weight: 700;
  line-height: 125%; /* 30px */
}

.preResultsDescription {
  margin-top: 16px;
  margin-bottom: 0;

  color: var(--color-slate-300, #9abed5);
  text-align: center;

  /* desktop/text-preset-4 */
  font-family: "Plus Jakarta Sans";
  font-size: var(--font-size-text-preset-4, 16px);
  font-style: normal;
  font-weight: 500;
  line-height: 150%; /* 24px */
}
