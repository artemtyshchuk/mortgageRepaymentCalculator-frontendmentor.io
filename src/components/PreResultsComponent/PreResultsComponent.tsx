import styles from "./PreResultsComponent.module.scss";
import { ReactComponent as PreResultIcon } from "assets/images/Layer_1.svg";

interface PreResultsComponentProps {}

export const PreResultsComponent = ({}: PreResultsComponentProps) => {
  return (
    <div className={styles.preResultsComponent}>
      <PreResultIcon />
      <p className={styles.preResultsTitle}>Results shown here</p>
      <p className={styles.preResultsDescription}>
        Complete the form and click “calculate repayments” to see what your
        monthly repayments would be.
      </p>
    </div>
  );
};
