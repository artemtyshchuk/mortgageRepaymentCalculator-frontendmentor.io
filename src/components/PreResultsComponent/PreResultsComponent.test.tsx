import { render, screen } from "@testing-library/react";
import { PreResultsComponent } from "./PreResultsComponent";

describe("PreResultsComponent", () => {
  it("should render component correctly", () => {
    render(<PreResultsComponent />);

    const preResultsComponent = screen.getByTestId("preResults");

    expect(preResultsComponent).toBeInTheDocument();
    expect(preResultsComponent).toMatchSnapshot();
  });
});
