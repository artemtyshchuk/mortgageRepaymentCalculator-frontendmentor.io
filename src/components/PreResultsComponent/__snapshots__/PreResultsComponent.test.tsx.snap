// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreResultsComponent should render component correctly 1`] = `
<div
  class="preResultsComponent"
  data-testid="preResults"
>
  <svg>
    Layer_1.svg
  </svg>
  <p
    class="preResultsTitle"
  >
    Results shown here
  </p>
  <p
    class="preResultsDescription"
  >
    Complete the form and click “calculate repayments” to see what your monthly repayments would be.
  </p>
</div>
`;
